docs/proposals/client-settings.md

```diff
@@ -0,0 +1,395 @@
+type KeepAliveTimeout struct {
+    // ServerTimeout sets the timeout during which a keep-alive client connection will stay open on the server side.
+    // The zero value disables keep-alive client connections.
+    // +optional
+    ServerTimeout *Duration `json:"serverTimeout,omitempty"`
+
+    // HeaderTimeout sets the timeout in the "Keep-Alive: timeout=time" response header field.
+    // +optional
+    HeaderTimeout *Duration `json:"headerTimeout,omitempty"`
+}
+
+// DisableType is the type of browsers to disable keep-alive connections on.
+type DisableType string
+
+var (
+    // NoneDisableType enables keep-alive connections for all browsers.
+    NoneDisableType DisableType = "None"
+
+    // SafariDisableType disables keep-alive connections for Safari and Safari-like browsers on macOS and macOS-like
+    // operating systems.
+    SafariDisableType DisableType = "Safari"
+
+    // MSIEDisableType disables keep-alive connections for old versions of MSIE, once a POST request is received.
+    MSIEDisableType DisableType = "MSIE"
+
+    // AllDisableType disables keep-alive connections for MSIE and Safari browsers.
+    AllDisableType DisableType = "All"
+)
+
+// Duration is a string value representing a duration in time.
+// The format is a subset of the syntax parsed by Golang time.ParseDuration.
+// Examples: 1h, 12m, 30s, 150ms.
+type Duration string
+
+// Size is a string value representing a size. Size can be specified in bytes, kilobytes (suffix k),
+// or megabytes (suffix m).
+// Examples: 1024, 8k, 1m.
+type Size string
+```
+
+### Versioning and Installation
+
+The version of the `ClientSettingsPolicy` API will be `v1alpha1`.
+
+The `ClientSettingsPolicy` CRD _may_ be installed by the Cluster Operator via Helm or with manifests. It will _not_ be required, and if the `ClientSettingsPolicy` CRD does not exist in the cluster, NGINX Gateway Fabric will not fail or log an error.
+
+### Status
+
+#### CRD Label
+
+According to the [Policy and Metaresources GEP](https://gateway-api.sigs.k8s.io/geps/gep-713/), the `ClientSettingsPolicy` CRD must have the `gateway.networking.k8s.io/policy: inherited` label to specify that it is an inherited policy.
+This label will help with discoverability and will be used by the planned Gateway API Policy [kubectl plugin](https://gateway-api.sigs.k8s.io/geps/gep-713/#kubectl-plugin-or-command-line-tool).
+
+#### Conditions
+
+According to the [Policy and Metaresources GEP](https://gateway-api.sigs.k8s.io/geps/gep-713/), the `ClientSettingsPolicy` CRD must include a `status` stanza with a slice of Conditions.
+
+The `Accepted` Condition must be populated on the `ClientSettingsPolicy` CRD using the reasons defined in the [PolicyCondition API](https://github.com/kubernetes-sigs/gateway-api/blob/main/apis/v1alpha2/policy_types.go). If these reasons are not sufficient, we can add implementation-specific reasons.
+
+#### Setting Status on Objects Affected by a Policy
```

Comment:
The GEP also has a provisional requirement that the GatewayClass also needs to have a status for all CRDs that are linked to the system it seems.

Reply:
It does, but unlike the Policy Affected status,  it's not well thought out yet. There's a separate [GEP](https://github.com/kubernetes-sigs/gateway-api/discussions/2118) for it that I think we should watch.

I can add a note about it to the future status section.